src/main/java/weavers/siltarae/mistake/dto/request/MistakeCreateRequest.java

```diff
@@ -0,0 +1,21 @@
+package weavers.siltarae.mistake.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
public보다는 protected가 나을 것 같은데 어떨까요? 
코드 일관성 부분에서도 그렇고, 접근 제한자는 빡빡하게 가져가는 게 좋지 않을까 하는 생각입니다!
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```